src/main/java/HW15_Multithreading/ArrayInitializer.java

```diff
@@ -0,0 +1,33 @@
+package HW15_Multithreading;
+
+import java.util.Arrays;
+
+public class ArrayInitializer {
+    public static void init(double[] array){
+        int length = array.length;
+        int mid = length/2;
+
+
+        var thread1 = new Thread(() -> {
+            for (int i = 0; i < mid; i++) {
+                array[i] = (array[i] * Math.sin(0.2 + i / 5.0) * Math.cos(0.2 + i / 5.0) * Math.cos(0.4 + i / 2.0));
+            }
+        });
+
+        var thread2 = new Thread(() -> {
+            for (int i = mid; i < length; i++) {
+                array[i] = (array[i] * Math.sin(0.2 + i / 5.0) * Math.cos(0.2 + i / 5.0) * Math.cos(0.4 + i / 2.0));
```

Comment:
```suggestion
               var j = i - mid;
                array[i] = (array[i] * Math.sin(0.2 + j / 5.0) * Math.cos(0.2 + j / 5.0) * Math.cos(0.4 + j / 2.0));
```

в такому випадку буде працювати формула: 
```
array[i] = array[n/2 + i];// (де n - розмір масиву)
```